python/src/opendp/extras/numpy/__init__.py

```diff
@@ -121,8 +120,8 @@ def array2_domain(
     if T is None:
         raise ValueError("must specify T, the type of data in the array")  # pragma: no cover
     T = dp.RuntimeType.parse(T)
-    if T not in ATOM_MAP:
-        raise ValueError(f"T ({T}) must be an elementary type")  # pragma: no cover
+    if T not in _PRIMITIVE_TYPES:
+        raise ValueError(f"T ({T}) must be a primitive type")  # pragma: no cover
```

Comment:
This is kind of a big change. I would have trouble saying whether ATOM_MAP or PRIMITIVE_TYPE is correct. Can you add test coverage to check?